### PR TITLE
barbican: update template to newton, remove not found configs, remove…

### PR DESCRIPTION
**Pull request by Itxaka**
… configs already on default values

### CODE REVIEW

**Comment by Itxaka** [comment]
[(5) Update templates for config files for Newton - Barbican](https://trello.com/c/6ozqOvAG/370-5-update-templates-for-config-files-for-newton-barbican)

**Comment by Itxaka** [comment]
rebuild with want_barbican=1 triggered

**Comment by Itxaka** [comment]
This works *but* barbican isnt working on the master branch anyway. So Im not sure this patch should go in until we can have barbican working on the master branch as to find if there is any issues with this patch

**Comment by Itxaka** [comment]
Depends on https://github.com/crowbar/crowbar-openstack/pull/686

**Comment by toabctl** [comment]
> Depends on #686

It is the same repo. Why are you not just adding the commits needed here to this PR? This would be less confusing imo.

**Comment by Itxaka** [comment]
@toabctl PR separation, plus I rather check if barbican is really working *before* this patch and *after* it instead of killing myself trying to see why the template change isnt working just to see that it has nothing to do with the template.

**Comment by Itxaka** [comment]
FYI: not using transport_url in barbican due to iot being broken: https://github.com/openstack/barbican/blob/stable/newton/etc/barbican/barbican.conf#L133

**Comment by Itxaka** [comment]
https://github.com/crowbar/crowbar-openstack/pull/686 was merged so tihs should be good to go after the reviews!

File: chef/cookbooks/barbican/templates/default/barbican.conf.erb [hunk 1]
```diff
@@ -2,13 +2,7 @@
```
Comment (toabctl) [inline chef/cookbooks/barbican/templates/default/barbican.conf.erb]: What about moving the config to the conf.d/ dir? Is that another commit?

Comment (Itxaka) [inline chef/cookbooks/barbican/templates/default/barbican.conf.erb]: wow, I thougth it was in there already o_O , will update the PR

Comment (Itxaka) [inline chef/cookbooks/barbican/templates/default/barbican.conf.erb]: ahh, that was already in: https://github.com/crowbar/crowbar-openstack/commit/ffc5168cbf672a514635ab22e19912b6a5399aba

**Pull request merged**